### DISCUSSION TITLE
Trello credential not workinh

### DISCUSSION THREAD

**Discussion by fabiofurlano** [langgenius/dify#5265] [General]
Hi, I know there have been some problems in the past which are now fixed with some API integrations. I have tested most of them, and they all work fine except for Trello. The credentials are not recognized. Of course, I have tried the usual troubleshooting, and I am sure the keys are right. Could you please help? I would really love to integrate Trello into my workflows.
thank you for your amazing work

**Comment by Yash-1511** [comment]
Hello @fabiofurlano ,

Could you please elaborate on your methodology for creating a key and token for Trello? Also, if you already possess a key and token, you can validate them by using the following API endpoint: `https://api.trello.com/1/members/me?key={api_key}&token={token}`.

**Comment by ghost** [comment]
https://github.com/langgenius/dify/discussions/5265#discussion-6823690

**Comment by fabiofurlano** [comment]
Sorry It was my mistake I was using the secret key insead of the api for the app:
It works a charm thanks and sorry for my mistake

**Comment by Yash-1511** [comment]
No problem 😃
